css-transitions-1/Overview.bs

```diff
@@ -920,14 +875,35 @@ Starting of transitions {#starting}
         property immediately changes to its final value).
       </p>
 
-      <p class="note">
-        Note that above rules mean that
-        transitions do not start when the computed
-        value of a property changes as a result of declarative animation
-        (as opposed to scripted animation).
-        This happens because the before-change style includes up-to-date
-        style for declarative animations.
-      </p>
+      <div class="example" id="commitStyles-no-transition-start">
+          <pre>
+&lt;style&gt;
+  #target {
+    transition: width 1s;
+    width: 100px;
+  }
+&lt;/style&gt;
+&lt;script&gt;
+  let animation = target.animate([{ "width": "200px" }], { duration: 1000 });
+  animation.finished.then(() => {
+    animation.commitStyles();
+    animation.cancel();
+  });
+&lt;/script&gt;
+          </pre>
+          <p>
+            The call to <code>commitStyles()</code>
+            will trigger a <a>style change event</a>,
+            with the <a href="#before-change-style">before-change</a>
+            <code>width</code> being <code>100px</code>,
+            and the <a href="#after-change-style">after-change</a>
+            <code>width</code> being <code>200px</code>.
```

Comment:
I don't think this is right? The style change event occurs before the inline style is updated so the `before-change` and `after-change` width at this point should both be 100px?